etc/profile-m-z/obsidian.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for obsidian-wayland
+# Description: Personal knowledge base and note-taking with Markdown files.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include obsidian-wayland.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/AMD
+noblacklist ${HOME}/.cache/mesa_shader_cache
+noblacklist ${HOME}/.cache/nvidia
+noblacklist ${HOME}/.local/share/applnk
+noblacklist ${HOME}/.local/share/vulkan
+noblacklist ${HOME}/.local/share/vulkan
+noblacklist ${HOME}/.config/kdedefaults
+noblacklist ${HOME}/.config/obsidian
+noblacklist ${HOME}/.config/vulkan
+
+whitelist ${HOME}/.cache/AMD
+whitelist ${HOME}/.cache/mesa_shader_cache
+whitelist ${HOME}/.cache/nvidia
+whitelist ${HOME}/.local/share/applnk
+whitelist ${HOME}/.local/share/vulkan
+whitelist ${HOME}/.local/share/vulkan
+whitelist ${HOME}/.config/kdedefaults
+whitelist ${HOME}/.config/obsidian
+whitelist ${HOME}/.config/vulkan
+
+ipc-namespace
+nonewprivs
+noroot
+protocol unix,inet,inet6
+#net none
+
+private-bin bash,basename,cat,cut,electron28,gawk,grep,obsidian,realpath,tr
```

Comment:
For ease-of-maintenance and improved future-proofing, instead of hardcoding a versioned electron we use
`electron,electron[0-9],electron[0-9][0-9]`
in private-bin.